packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -102,6 +102,9 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement | Focu
 
   /** Allow focus to move to root */
   allowFocusRoot?: boolean;
+
+  /** Allows tab key to be handled, a side effect is that users will not be able to tab out of the focus zone */
```

Comment:
Unless they are at the first or last element, right?

Reply:
No, we'll just repeat through the contextual menu item. 

From talking with @jspurlin and @betrue-final-final the conclusion seems to be that for now this will be the desired effect for now because it's what's being done in the the Word rich client and there will be further discussions in the future, about what the desired result should be.

Reply:
this appears to be breaking our ability to tab from one focus zone to another. Once we get into a focus zone with a contextual menu, our tabs do nothing bug cycle through the items in the menu. Also, it appears to capture tabs even if the menu is not opened.